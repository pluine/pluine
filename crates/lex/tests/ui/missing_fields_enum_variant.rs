#[derive(pluine_lex_macros::Spanned)]
enum Foo<'src> {
    A(),
    B(&'src str),
}

fn main() {}
