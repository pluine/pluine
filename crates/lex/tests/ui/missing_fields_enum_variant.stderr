error: no unnamed field found in enum variant which may implement `Span`
 --> tests/ui/missing_fields_enum_variant.rs
  |
  |     A(),
  |      ^^
